internal/updater/updater.go

```diff
@@ -72,36 +73,54 @@ func NewAvailableUpdate(channel, version, platform, path, sha256, tag string) *A
 	}
 }
 
+func NewAvailableUpdateFromGraph(au *graph.AvailableUpdate) *AvailableUpdate {
+	if au == nil {
+		return &AvailableUpdate{}
+	}
+	return NewAvailableUpdate(au.Channel, au.Version, au.Platform, au.Path, au.Sha256, "")
+}
```

Comment:
Nit: Why add a dependency on `graph` here? Seems whatever is calling this should just be calling `NewAvailableUpdate()`.

Reply:
This is added as a convenience for callers so that they don't have to extract the gql response fields themselves as in line 80. It also provides simple nil handling.